README.md

```diff
@@ -3,34 +3,62 @@
 > A Python wrapper for using the [ocr.space API](https://ocr.space/ocrapi).
 
 ## Installation
+
 Simply install from `pip`:
+
 ```sh
 pip install ocrspace
 ```
 
 ## Use
+
+First, get an API key from [Free OCR API](https://ocr.space/OCRAPI), otherwise the default apikey ```helloworld``` will be used, which is [severely rate limited.](https://ocr.space/faq#span12)
+
 First you'll need to import and instantiate the API wrapper:
-```py
+
+```python
+import ocrspace
+
+api_key = 'apikey retrieved from Free OCR API'
+
+api = ocrspace.API(api_key=api_key)
+# Or if you have a custom API host, API key or desired language, pass those:
+api = ocrspace.API(endpoint='https://example.host', api_key=api_key, language=ocrspace.Language.Croatian)
+```
+
+If you wish to change the OCR engine used, you'll have to import the enum class Engine from ocrspace and pass the value of Engine.ENGINE_2 to the api instantiation. By default it uses Engine.val_1:
+
+```python
 import ocrspace
-api = ocrspace.API()
+from ocrspace import Engine
+
+api_key = "apikey retrieved from Free OCR API"
+
+api = ocrspace.API(api_key=api_key, engine=Engine.ENGINE_2)
 # Or if you have a custom API host, API key or desired language, pass those:
-api = ocrspace.API(endpoint='https://example.host', api_key='Insert key here', language=ocrspace.Language.Croatian)
+api = ocrspace.API(endpoint='https://example.host', api_key=api_key, language=ocrspace.Language.Croatian, engine=Engine.ENGINE_2)
```

Comment:
Since we're using `ocrspace.Language`, it makes sense to me to use `ocrspace.Engine` as well rather than importing directly.

Reply:
updated on commit https://github.com/ErikBoesen/ocrspace/commit/dbcf91afd16977d60018f2985188080ed8b987ee